website/docs/guides/provider_reference.html.markdown

```diff
@@ -136,6 +136,15 @@ by precedence.
     the path of your service account key file in the
     `GOOGLE_APPLICATION_CREDENTIALS` environment variable, or configure
     authentication through one of the following;
+  
+     ```hcl
+     provider "google" {
+         alias       = "other"
+         project     = var.my_other_project_id
```

Comment:
Please change the values in all of these fields to be example text rather than references to variables. It will be consistent with other config snippets in this doc